qml/pages/FirstPage.qml

```diff
@@ -23,6 +24,46 @@ Page {
                            { gpsDataSource: page.gpsDataSource, compass: page.compass})
         }
     }
+    Item {
+            id: gpsTimes
+            property date activateGPSTimestamp: new Date()
+            property date firstFixTimestamp: new Date()
+            property bool pendingFix : true
+            property int secsSincePosition : Math.round((new Date() - positionSource.position.timestamp)/1000)
+
+            property int secs2FF : 0
+            property string secs2FFStr: roundElapsedTime(secs2FF)
+            property string tTFF: {
```

Comment:
I don't have a clue what "FF" means in this context (maybe it becomes apparent later on?) so I may rename them before merging.

Reply:
FirstFix.  I will rename.
I am moving this timing into providers.qml, so it is visible to cover.qml also.